app/pulp/app/models/task.py

```diff
@@ -27,7 +27,7 @@ class ReservedResource(Model):
     resource = models.TextField()
 
     task = models.OneToOneField("Task")
-    worker = models.ForeignKey("Worker")
+    worker = models.ForeignKey("Worker", on_delete=models.CASCADE, related_name="reservations")
```

Comment:
I saw @seandst use this model-level setting which is very likable: https://docs.djangoproject.com/en/1.8/ref/models/options/#django.db.models.Options.default_related_name